lgtm.yml

```diff
@@ -0,0 +1,6 @@
+# Configure LGTM for this package
+
+extraction:
+  python:  # Configure Python
+    python_setup:  # Configure the setup
+      version: 3  # Specify Version 3
```

Comment:
Can we also add path classifiers:
```
path_classifiers:
  test:
    - **/tests/**
```

Not sure how to project out `versioneer`, but it would be good to do so if possible.

Reply:
It looks like LGTM already has that filter for `test` (at least according to the browser, everything in the various `tests` folders and matching `*test*` are tagged correctly and not looked at.) Unless you can give me a compelling reason, I am not going to add that for the sake of file simplicity and I'll figure out a versioneer fix.